auditorium/gulpfile.js

```diff
@@ -46,32 +51,43 @@ function createLocalizedBundle (locale) {
   return gulp.parallel(scriptTask, vendorTask)
 }
 
-function makeScriptTask (dest, locale) {
+function makeScriptTask(dest, locale) {
   return function () {
     var transforms = JSON.parse(JSON.stringify(pkg.browserify.transform))
     // we are setting this at process level so that it propagates to
     // dependencies that also require setting it
     process.env.LOCALE = locale
+    process.env.SCRIPT_INTEGRITY_HASH = "unkown"
     var b = browserify({
-      entries: './index.js',
-      // See: https://github.com/nikku/karma-browserify/issues/130#issuecomment-120036815
-      postFilter: function (id, file, currentPkg) {
-        if (currentPkg && currentPkg.name === pkg.name) {
-          currentPkg.browserify.transform = []
-        }
-        return true
-      },
-      transform: transforms.map(function (transform) {
-        if (transform === '@offen/l10nify' || (Array.isArray(transform) && transform[0] === '@offen/l10nify')) {
-          return ['@offen/l10nify']
-        }
-        if (transform === 'envify' || (Array.isArray(transform) && transform[0] === 'envify')) {
-          return ['envify', { LOCALE: locale }]
-        }
-        return transform
+        entries: './index.js',
+        // See: https://github.com/nikku/karma-browserify/issues/130#issuecomment-120036815
+        postFilter: function (id, file, currentPkg) {
+          if (currentPkg && currentPkg.name === pkg.name) {
+            currentPkg.browserify.transform = []
+          }
+          return true
+        },
+        transform: transforms.map(function (transform) {
+          if (transform === '@offen/l10nify' || (Array.isArray(transform) && transform[0] === '@offen/l10nify')) {
```

Comment:
It would be good if you could undo these formatting only changes as well even if standard is ok with them. It'll be easier to understand what changed if we don't have the additional noise in the diff.